.github/workflows/ci.yml

```
@@ -0,0 +1,59 @@
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['8.1', '8.2', '8.3', '8.4']
+        symfony_constraint: ['^5.0', '^6.0', '^7.0']
+        prefer_stability: ['--prefer-lowest', '--prefer-stable']
+
+      exclude:
```

actionlint (1.7.7)

unexpected key "exclude" for "strategy" section. expected one of "fail-fast", "matrix", "max-parallel"

(syntax-check)